### DISCUSSION TITLE
Media Library

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1236] [Q&A (Help)]
Does anyone have a demo of a realistic implementation of Spatie MediaLibrary where a user can manage his own gallery and can choose images from previews? :)

[a comment by an automated account is not shown]

**Comment by karandatwani92** [comment]
Hey @blondie63 

Please help me understand why you would choose the Spatie MediaLibrary `gallery` over the `browse` & `browse_multiple` fields for the same thing.

Right now, I do not see such a thing in the plan. If you are making one, you can choose to create it in a package form and share it with the Backpack community.

--
Just In case you are looking for an example of working with Spatie MediaLibrary, here is one in the demo.
- https://demo.backpackforlaravel.com/admin/product/210/edit#media
- [ProductCrudController](https://github.com/Laravel-Backpack/demo/blob/41681217933e4dbffa3d0ff282595f3b8c5c7b16/app/Http/Controllers/Admin/ProductCrudController.php#L210) 
- [Product Model](https://github.com/Laravel-Backpack/demo/blob/main/app/Models/Product.php)

**Reply by blondie63** [reply]
Hi @karandatwani92, i'm doing an icon/images management for supervisor for shared to all user
Users will be able to choose system or personal icons/images via upload
As soon as I finish I'll show it to you
